core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/CsvTests.kt

```diff
@@ -1,366 +0,0 @@
-package org.jetbrains.kotlinx.dataframe.io
-
-import io.kotest.assertions.throwables.shouldNotThrowAny
-import io.kotest.matchers.nulls.shouldNotBeNull
-import io.kotest.matchers.shouldBe
-import kotlinx.datetime.LocalDateTime
-import org.apache.commons.csv.CSVFormat
-import org.intellij.lang.annotations.Language
-import org.jetbrains.kotlinx.dataframe.DataFrame
-import org.jetbrains.kotlinx.dataframe.api.ParserOptions
-import org.jetbrains.kotlinx.dataframe.api.allNulls
-import org.jetbrains.kotlinx.dataframe.api.convert
-import org.jetbrains.kotlinx.dataframe.api.dataFrameOf
-import org.jetbrains.kotlinx.dataframe.api.group
-import org.jetbrains.kotlinx.dataframe.api.groupBy
-import org.jetbrains.kotlinx.dataframe.api.into
-import org.jetbrains.kotlinx.dataframe.api.isEmpty
-import org.jetbrains.kotlinx.dataframe.api.schema
-import org.jetbrains.kotlinx.dataframe.api.toStr
-import org.jetbrains.kotlinx.dataframe.ncol
-import org.jetbrains.kotlinx.dataframe.nrow
-import org.jetbrains.kotlinx.dataframe.testCsv
-import org.jetbrains.kotlinx.dataframe.testResource
-import org.junit.Test
-import java.io.File
-import java.io.StringReader
-import java.io.StringWriter
-import java.net.URL
-import java.util.Locale
-import kotlin.reflect.KClass
-import kotlin.reflect.typeOf
-
-@Suppress("ktlint:standard:argument-list-wrapping")
-class CsvTests {
-
-    @Test
-    fun readNulls() {
-        val src =
-            """
-            first,second
-            2,,
-            3,,
-            """.trimIndent()
-        val df = DataFrame.readDelimStr(src)
-        df.nrow shouldBe 2
-        df.ncol shouldBe 2
-        df["first"].type() shouldBe typeOf<Int>()
-        df["second"].allNulls() shouldBe true
-        df["second"].type() shouldBe typeOf<String?>()
-    }
-
-    @Test
-    fun write() {
-        val df = dataFrameOf("col1", "col2")(
-            1, null,
-            2, null,
-        ).convert("col2").toStr()
-
-        val str = StringWriter()
-        df.writeCSV(str)
```

Comment:
Sorry, where all these tests (with this logic) was moved or adopted for the Deephaven csv?

Reply:
exactly, it's in DelimCsvTsvTests in dataframe-csv